.taskcat.yml

```diff
@@ -0,0 +1,23 @@
+project:
+  name: quickstart-cloudbees-core
+  owner: dschott@cloudbees.com
+  package_lambda: false
+  regions:
+  - us-east-1
+  - us-east-2
+  - us-west-2
+  - eu-west-1
+  s3_bucket: ''
+tests:
+  cloudbees-core:
+    parameters:
+      AvailabilityZones: $[taskcat_genaz_3]
+      KeyPairName: override
+      KubernetesVersion: '1.12'
```

Comment:
we should probably bump this to at least 1.14

Reply:
Good call.  Looks like it's available:
https://github.com/aws-quickstart/quickstart-amazon-eks/blob/master/templates/amazon-eks.template.yaml#L140-L143

In my branch I removed this parameter:
https://github.com/schottsfired/quickstart-cloudbees-core-1/commit/2f71fb74652c5b714285598728ad9653d2622109#diff-bab7f2d0cbf444f8667350f620b9a278L16

So that it'd use the default:
https://github.com/schottsfired/quickstart-cloudbees-core-1/blob/develop/templates/cloudbees-core-master.template.yaml#L798

Reply:
1.14: https://github.com/cloudbees/quickstart-cloudbees-core/pull/7